roles/flag-lifecycle/all-environments.tf

```diff
@@ -15,9 +18,9 @@ output "project-roles"{
 /// Read only access to a subset of projects
 /// May create approval requests.
 /// Rationale: Empower any team to request changes to the application. May be scoped with a tag. Think requesting user impersonation
-resource launchdarkly_custom_role "view-project" {
```

Comment:
Do we like member better than view project? I'm fine with either one. Reason I had changed it to view is because it seemed clearer in the UI